nixos/modules/services/cluster/kubernetes/addons/dns.nix

```diff
@@ -151,20 +190,7 @@ in {
           namespace = "kube-system";
         };
         data = {
-          Corefile = ".:${toString ports.dns} {
-            errors
-            health :${toString ports.health}
-            kubernetes ${cfg.clusterDomain} in-addr.arpa ip6.arpa {
-              pods insecure
-              fallthrough in-addr.arpa ip6.arpa
-            }
-            prometheus :${toString ports.metrics}
-            forward . /etc/resolv.conf
-            cache 30
-            loop
-            reload
-            loadbalance
-          }";
```

Comment:
It doesn't seem to make any difference and the tests pass in both cases.  I also tried `''` strings in my live deployment and that worked as well.

I changed this PR to use `''` since that seems to be the common choice for config files and it removes some unnecessary indentation in the resulting file.